https://bugs.gentoo.org/413547

From 3b98b99eaa23f242b984af8cbf4a0e707bc7eae8 Mon Sep 17 00:00:00 2001
From: Corinna Vinschen <corinna@vinschen.de>
Date: Thu, 23 Feb 2012 09:30:17 +0000
Subject: [PATCH] 	* configure.host: Accept "arm*" host spec instead of
 just "arm".

---
 newlib/ChangeLog      |    4 ++++
 newlib/configure.host |    8 ++++----
 2 files changed, 8 insertions(+), 4 deletions(-)

+2012-02-23  Dan Egnor  <egnor@ofb.net>

	* configure.host: Accept "arm*" host spec instead of just "arm".

diff --git a/newlib/configure.host b/newlib/configure.host
index a6673af..dc44e80 100644
--- a/newlib/configure.host
+++ b/newlib/configure.host
@@ -98,7 +98,7 @@ case "${host_cpu}" in
   arc)
 	machine_dir=
 	;;
-  arm)
+  arm*)
 	machine_dir=arm
 	;;
   avr*)
@@ -367,7 +367,7 @@ case "${host}" in
   arc-*-*)
 	sys_dir=arc
 	;;
-  arm-*-*)
+  arm*-*-*)
 	sys_dir=arm
 	if [ "x${newlib_may_supply_syscalls}" = "xno" ] ; then
 	  have_crt0="no"
@@ -560,10 +560,10 @@ case "${host}" in
   arc-*-*)
 	syscall_dir=syscalls
 	;;
-  arm-*-pe)
+  arm*-*-pe)
 	syscall_dir=syscalls
 	;;
-  arm-*-*)
+  arm*-*-*)
 	syscall_dir=syscalls
 # If newlib is supplying syscalls, select which debug protocol is being used.
 # ARM_RDP_MONITOR selects the Demon monitor.
-- 
1.7.8.5

